theme/templates/email/signup_verify.txt

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% block main %}
+{% trans "Welcome to HydroShare. This email address was used to request an account on HydroShare." %}
+{% trans "If you originated the request, please use the link below to verify your email address and activate your account." %}
+http://{{ request.get_host }}{{ verify_url }}
```

Comment:
This is customized on top of Mezzanine's default template which we are currently using. Just tried to register a new account on https://dev.hydroshare.org, and the link I received in the verification email starts with http like http://dev.hydroshare.org/accounts/verify/... which once clicked, got redirected to https://dev.hydroshare.org. So it looks like it is working on HTTPS production environment. Not sure whether there is any security implication though.